src/SqlAsyncCollector.cs

```diff
@@ -73,6 +73,7 @@ internal class SqlAsyncCollector<T> : IAsyncCollector<T>, IDisposable
 
         private readonly List<T> _rows = new List<T>();
         private readonly SemaphoreSlim _rowLock = new SemaphoreSlim(1, 1);
+        private static ServerProperties _serverProperties;
```

Comment:
This shouldn't need to be static - and would cause problems if there's multiple functions running in the same host but connecting to different servers.